ffi/dotnet-idsdk/IoticsIdLib/src/Identity.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System.Net;
+
+namespace IOTICS;
+
+// Custom RuntimeException class
+public class IoticsLibException(string message) : ApplicationException(message)
+{
+}
+
+public class DelegationException(string message) : IoticsLibException(message) {
+    
+}
+
+/** 
+Factory class for identity. 
+It takes a resolver address as discovered by https://<your_space>.iotics.space/index.json
+
+This class only exposes a fraction of the methods of the identity library, only the most common for beginners and specifically those defined here using ffi and C-style bindings is here
+ https://github.com/Iotic-Labs/iotics-identity-go/blob/v3/ffi/ffi_wrapper.go  as wrappers for FFI and C-style bindings.
+
+The full spec of the identity is here https://drive.google.com/file/d/1nlJxB08cpYStcunMyhXjOvHStWZEv4ch/view
+
+The full API is here https://github.com/Iotic-Labs/iotics-identity-go/tree/v3/pkg/api
+
+A simple usage of the API is in the file Main.cs
+
+*/
+public class Identity(string resolverAddress)
+{
+    public string ResolverAddress
+    {
+        get
+        {
+            return Tools.IsUrl(resolverAddress) ? Tools.RemoveTrailingSlash(resolverAddress) : resolverAddress;
+        }
+    }
+
+
+    /// <summary>
```

Comment:
It's nice to have these doc strings. However - I can see that the Java one doesn't have them and of course it would be nice not to duplicate them. Should we instead not have them here at all and document them in `ffi_wrapper.go` with at most links to the corresponding functions?

Reply:
No - a C# developer will need to look at C# code.
Java isn't delivered to external developers so there's less emphasis on making it prettier.

Reply:
Right.. so in the future we'll be maintaining doc-strings (which should all the same thing) in rust, Java & C#. (I guess that's unavoidable.)

Good point about Java - should we mark that as "beta" / "unsupported" then? (The repo after all is public - how would people distinguish what is ready and what is not? :thinking: )